nix-test/src/const.c

```diff
@@ -209,6 +209,31 @@ get_int_const(const char* err) {
     GET_CONST(ECANCELED);
 #endif
 
+#if defined(__OpenBSD__)
```

Comment:
I added these to get the errno tests passing. It looks like the [check_errno macro](https://github.com/wezm/nix/blob/11dae339cf8111ff326ecb3d6763c201e7aba757/src/errno.rs#L1920-L1924) uses them via [assert_const_eq](https://github.com/wezm/nix/blob/11dae339cf8111ff326ecb3d6763c201e7aba757/nix-test/src/lib.rs#L17-L27) or did I miss something?